src/imputation/tmi_imputation.py

```diff
@@ -187,17 +185,17 @@ def sort_df(target_variable: str, df: pd.DataFrame) -> pd.DataFrame:
 def apply_trim_check(
     df: pd.DataFrame,
     variable: str,
-    check_value=10,
-) -> pd.DataFrame:  # TODO add check_value to a config
+    trim_threshold=10,
```

Comment:
Should `trim_threshold` be abstracted into the config?

Reply:
Or if it's been abstracted elsewhere then might need to delete this

Reply:
It has been added to the config, it's passed from the calling function. We could delete the default value though.